### PR TITLE
Kubelet Service bind-mount Prompts for Flexvolume

### PR DESCRIPTION
**Pull request by dahendel**
Continuation of #474

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -309,6 +309,14 @@ services:
       - "/usr/libexec/kubernetes/kubelet-plugins:/usr/libexec/kubernetes/kubelet-plugins:z"
 ```
 
+## FlexVolume Binds
+When prompted to `Enable the FlexVolume driver mount [n]: ` if the user answers yes, then rke will also prompt for the flexvolume bind mount path.
```
Comment (galal-hussein) [inline README.md]: Can you explain that this is related to `rke config`

File: cmd/config.go [hunk 2]
```diff
@@ -313,6 +315,48 @@ func getServiceConfig(reader *bufio.Reader) (*v3.RKEConfigServices, error) {
 		return nil, err
 	}
 	servicesConfig.Kubelet.InfraContainerImage = infraPodImage
+
+	// Add the flexvolume mount to kublet ExtraBinds
+	enableFlexVolMount, err := getConfig(reader, "Enable the FlexVolume driver mount", "no")
+	if err != nil {
+		return nil, err
+	}
+
```
Comment (galal-hussein) [inline cmd/config.go]: can you remove the extra new lines between the code lines

File: cmd/config.go [hunk 3]
```diff
@@ -313,6 +315,48 @@ func getServiceConfig(reader *bufio.Reader) (*v3.RKEConfigServices, error) {
...
+	}
+
+	// Add additional ExtraBinds
+	additionalBinds, err := getConfig(reader, "Add additional bind mounts for kubelet service", "no")
+	if err != nil {
+		return nil, err
+	}
+
+	if strings.ContainsAny(additionalBinds, yesAnswer) {
```
Comment (galal-hussein) [inline cmd/config.go]: If you are going to use yesAnswer, can you please change it in the rest of the questions in the rke config

**Comment by dahendel** [comment]
@galal-hussein Changes made, I can fix conflicts if that's all I need to do now.

**Comment by dahendel** [comment]
@galal-hussein @superseb Should I close this? It looks like the problem I was looking to solve was solved in other areas #800

**Pull request closed without merging**